packages/angle-headers.cmake

```
@@ -1,11 +1,18 @@
 ExternalProject_Add(angle-headers
-    URL https://gitlab.com/shinchiro/angle/-/archive/main/angle-main.zip?path=include
+    GIT_REPOSITORY https://github.com/google/angle.git
+    SOURCE_DIR ${SOURCE_LOCATION}
+    GIT_CLONE_FLAGS "--sparse --filter=tree:0"
+    GIT_REMOTE_NAME origin
+    GIT_TAG main
+    GIT_CLONE_POST_COMMAND "sparse-checkout set include/EGL include/KHR"
```

```suggestion
    GIT_CLONE_FLAGS "--sparse --filter=tree:0"
    GIT_CLONE_POST_COMMAND "sparse-checkout set include/EGL include/KHR"
```
Moved GIT_CLONE_FLAGS to just above GIT_CLONE_POST_COMMAND